### PR TITLE
added Policy as code tutorial link

### PR DESCRIPTION
**Pull request by raiyanalam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ Check here for a current list of [**known issues**](#known-issues) for Azure Pol
 - [How to assign policies using the Azure portal](https://docs.microsoft.com/azure/governance/policy/assign-policy-portal)
 - [How to assign policies using Azure PowerShell](https://docs.microsoft.com/azure/governance/policy/assign-policy-powershell)
 - [How to assign policies using Azure CLI](https://docs.microsoft.com/azure/governance/policy/assign-policy-azurecli)
+- [Export and manage Azure policies as code with GitHub](https://docs.microsoft.com/en-us/azure/governance/policy/tutorials/policy-as-code-github)
```
Comment (robga) [inline README.md:31]: Please remove 'en-us' from the url

**Review by robga — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (raiyanalam) [reply README.md]: Done. Thanks for catching this 👍

**Review by DCtheGeek — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (raiyanalam) [reply README.md]: Updated to "Azure Policy resources". Thanks! - @DCtheGeek

**Review by robga — APPROVED** [review]

**Comment by raiyanalam** [comment]
@DCtheGeek : Please review and merge.

**Review by DCtheGeek — APPROVED** [review]

**Pull request merged**